packages/query/src/graphql/plugins/PgDictionaryPlugin.ts

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {QueryBuilder} from 'graphile-build-pg';
+import {makeExtendSchemaPlugin, gql} from 'graphile-utils';
+
+const SORT_ARGS = {
+  lessThan: '<',
+  lessThanOrEqualTo: '<=',
+  greaterThan: '>',
+  greaterThanOrEqualTo: '>=',
+};
+
+const generateArgs = (queryBuilder: QueryBuilder, args, sql) => {
```

Comment:
- [ ] I can make this cleaner

Reply:
You can use `builder.hook("GraphQLObjectType:fields:field:args",` like in here https://github.com/graphile-contrib/postgraphile-plugin-connection-filter/blob/fda1bc1e4474a3a64be10f3228c81b88560f6799/src/PgConnectionArgFilterPlugin.ts#L27